python/ray/tune/suggest/nevergrad.py

```diff
@@ -111,6 +124,15 @@ def __init__(self,
         self._opt_factory = None
         self._nevergrad_opt = None
 
+        if points_to_evaluate is None:
+            self._points_to_evaluate = None
+        elif not isinstance(points_to_evaluate, Sequence):
+            raise ValueError(
+                f"Invalid object type passed for `points_to_evaluate`: {type(points_to_evaluate)}. "
+                f"Please pass a list of points (dictionaries) instead.")
+        else:
+            self._points_to_evaluate = points_to_evaluate
```

Comment:
We will either have to convert to a list here (i.e. `self._points_to_evaluate = list(points_to_evaluate)`) or check for `MutableSequence` above, as this implements the `pop()` interface we use in the code below.

Reply:
done